lib/vintage_net_wizard/web/router.ex

```diff
@@ -220,21 +220,33 @@ defmodule VintageNetWizard.Web.Router do
     case status = BackendServer.configuration_status() do
       :good ->
         %{
-          value: status,
+          value: configuration_status_to_string(status),
           class: "text-success",
           title: "Device successfully connected to a network in the applied configuration"
         }
 
       :bad ->
         %{
-          value: status,
+          value: configuration_status_to_string(status),
           class: "text-danger",
           title:
             "Device was unable to connect to any network in the configuration due to bad password or a timeout while attempting."
         }
 
       _ ->
-        %{value: status, class: "text-warning", title: "Device waiting to be configured."}
```

Comment:
This is intended to be a comment on line 236. The `_` can be `:not_configured` since the spec for `BackendServer.configuration_status/0` is `:not_configured | :good | :bad`.